l10n/templates/strings.properties

```diff
@@ -65,6 +68,10 @@ search_header={search_engine_name} Search
 search_web_placeholder=Search the Web
 search_settings=Change Search Settings
 
+# LOCALIZATION NOTE (section_info_option): This is the screenreader only text
```

Comment:
I'm not sure I understand this comment, besides the fact that the text is used by screenreaders. What’s the "section helper text"?